packages/terra-base/package.json

```diff
@@ -22,7 +22,8 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "react-dom": "^16.4.2",
+    "react-intl": "^2.4.0"
```

Comment:
We should bump to ^2.8.0

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2288/commits/76423fcbbd7d3f2301452db3667e6f69db6960f0